src/css/styles.scss

```diff
@@ -1,46 +1,90 @@
-// Load CSS Reset from NPM
-// CSS resets "normalize" how browser styles are applied.
-// Try uncommenting this and saving. What changes in the browser?
-@import "~normalize.css";
+.top {
+	display: block;
+	height: 80px;
+	margin-left: auto;
+	margin-right: auto;
+	background-color: #f8f9eb;
+	font-family: raleway light;
+	color: #052a3d;
+	font-size: 26px;
+  text-align: center;
+  padding: 10px 0;
+}
 
-// An example of a partial.
-// Check out the _typography.scss file to edit the CSS inside!
-@import "typography";
-@import "layout";
+.dropbtn {
+  background-color: #f8f9eb;
+  color: #052a3d;
+  padding: 10px;
+  font-size: 10px;
+  border: none;
+}
 
-// You can still write styles in this file too.
-// Here's an example of how you could write media queries more easily with Sass.
+/* The container <div> - needed to position the dropdown content */
+.dropdown {
+  float:right;
+}
 
-$large-phone: 500px;
-$tablet-vertical: 768px;
-$desktop: 1200px;
+/* Dropdown Content (Hidden by Default) */
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #f8f9eb;
+  font-family: raleway light;
+  font-size:10px;
+  min-width: 50px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+}
 
-// And some color variables.
-// Look how much easier it is to keep track of what these are than hex codes!
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {background-color: #ddd;}
 
-$red: #ab424a;
-$orange: #dd7c00;
-$green: #3b975f;
-$blue: #4261ab;
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content {display: block;}
 
-.page-heading {
-	color: $red;
+/* Change the background color of the dropdown button when the dropdown content is shown */
+.dropdown:hover .dropbtn {background-color: pink;}
 
-	// Sass lets you write your media queries inline if you want.
-	// This is why I'm always forgetting I have to put the class inside
-	// the media query in regular CSS. I never have to in Sass!
-	// I like combining these with variables to make it easier to read
-	// my media queries.
+.bgcolor {
+	background-color: #f8f9eb;
+	margin-top: 1px;
+  padding: 5px 0;
+	color: #052a3d;
+}
+
+.title1{
+  font-family: raleway light;
+	color: #052a3d;
+	font-size: 30px;
+  margin-left: 3%;
+  margin-top: 60px;
+  text-align:center;
+}
 
-	@media screen and (min-width: $large-phone) {
-		color: $orange;
-	}
 
-	@media screen and (min-width: $tablet-vertical) {
-		color: $green;
-	}
+.pic1{
+  width:50%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  border: 3px solid #052a3d;
+}
 
-	@media screen and (min-width: $desktop) {
-		color: $blue;
-	}
+.pic{
+  text-align:center;
+  margin-top:0px;
+  border-style: solid;
+  border-color: #052a3d;
+  font-familt:raleway;
```

Comment:
There's a typo here and your font family is different than the ones above - double check this